persistence/sql/migrations/sql/20220825134336000001_not_null_constraint_verification_token_flow_id.sqlite3.down.sql

```diff
@@ -0,0 +1,37 @@
+ALTER TABLE identity_verification_tokens
+RENAME TO identity_verification_tokens_;
+
+CREATE TABLE "identity_verification_tokens" (
```

Comment:
You could probably make this a bit easier with:

```
ALTER TABLE ... ADD COLUMN old NOT NULL
UPDATE ... SET new = old
ALTER TABLE DROP COLUMN old
ALTER TABLE RENAME COLUMN new, old
CREATE INDEX ...
```

Reply:
just make sure to add the indices correctly :)

Reply:
Oh, or is it not possible to add foreign keys like this?

Reply:
nope, should be possible:

https://www.sqlite.org/lang_altertable.html


```
ALTER TABLE ... ADD COLUMN old NOT NULL REFERENCES ...
```

Reply:
You can't add foreign keys to an existing table in SQLite, apparently. I'd prefer your solution too, but that isn't possible AFAICT, since `selfservice_verification_flow_id` is an FK.

Reply:
Huh, interesting. Then I had a different issue, I guess. The error messages of SQLite aren't that helpful ^^ Thanks, I'll try that.

Reply:
Frustrating stuff: Adding a foreign key works, removing the old one doesn't. That's why we apparently need to copy the whole table... Dropping the (nullable) `selfservice_verification_flow_id` column leaves behind its foreign key, and SQLite complains about that: `error in table identity_verification_tokens after drop column: unknown column "selfservice_verification_flow_id_" in foreign key definition`

Reply:
Oh, I see ... damn :(